Makefile

```diff
@@ -96,6 +99,9 @@ release: ## Build release binaries
 		done ; \
 	done
 
+	$(KUBECTL) create -f config --dry-run=client -o yaml | \
+	  $(SED) 's|ko://github.com/triggermesh/triggermesh/cmd/\(.*\)|$(IMAGE_REPO)/\1:${IMAGE_TAG}|' > $(DIST_DIR)/triggermesh-core.yaml
```

Comment:
Since the YAML manifests are generated from the `config/` directory, the generated manifest won't include the `triggermesh` namespace creation. The namespace was removed from the configs to safeguard us from unintentionally deleting the `triggermesh` namespace.

Without the namespace in the config, the install guide would require that the user first create the `triggermesh` namespace.

Reply:
we could have the namespace creation somewhere in config and have a kustomize file, then issue a
```
$(KUBECTL)  create -k config/release --dry-run=client ...
```
or maybe remove the samples from config, add a namespace subfolder and issue a
```
$(KUBECTL) create -Rf config --dry-run=client ...
```

Reply:
@odacremolbap does this make sense? https://github.com/triggermesh/triggermesh/pull/108/commits/01350dc44d38806decbceba558e0c661b041dd37

Reply:
i can confirm that `ko` does not bother about the contents of `config/namespace`

Reply:
I liked the idea of making the namespace part of a kustomization, but I guess the subdirectory works too. Neither `ko` nor `kubectl` recurse by default.